website/content/about/CloudNativeSustainabilityWeek.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
Thanks for creating this website page, @leonardpahlke, it's taking a really good shape! I have a few suggestions to the content below.

I have an overall suggestion as well: this may be one of our highlight events this year and I think it would be valuable to make it more visible when people access the TAG ENV website. As of now it's located in the "About" section which I don't feel is quite a suitable place for this specific page since it's also related to an event. Can we either make a separate tab in the nav menu of the website, side by side with "About" or put it under "Events" where I feel it would be a more natural place for this kind of content to reside? 😊

Reply:
Agree, let move it!